### PR TITLE
Labs api setup

### PR DESCRIPTION
**Pull request by armandoroman1016**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: mission-control-client/src/actions/personActions.js [hunk 1]
```diff
@@ -2,24 +2,129 @@ import axiosLabsGraphQL from '../utils/axiosLabsGraphQL'
...
+                dispatch({type: GET_PERSON_SUCCESS, payload: res.data})
+            })
+            .catch( err => {
+                dispatch({type: GET_PERSON_ERROR, payload: err.response})
+                console.log(err)
+            })
+    }
+}
+
+// TODO EMPTY DATASET NEEDS WORK
```
Comment (dionys93) [inline mission-control-client/src/actions/personActions.js:48]: #20 Empty datasets to look into

File: mission-control-client/src/actions/personActions.js [hunk 2]
```diff
@@ -2,24 +2,129 @@ import axiosLabsGraphQL from '../utils/axiosLabsGraphQL'
...
+                dispatch({type: GET_PEOPLE_GROUP_SUCCESS, payload: res.data})
+            })
+            .catch( err => {
+                console.log(err)
+                dispatch({type: GET_PEOPLE_GROUP_ERROR, payload: err.response})
+            })
+    }
+}
+
+// TODO EMPTY DATASET NEEDS WORK - NEEDS TO BE TESTED
```
Comment (dionys93) [inline mission-control-client/src/actions/personActions.js:69]: #20 Empty datasets

File: mission-control-client/src/actions/projectActions.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            .then(res => {
+                const project = res.data.data.project;
+                console.log(project)
+                dispatch({ type: GET_PROJECT_SUCCESS, payload: project })
+            })
+            .catch(err => dispatch({ type: GET_PROJECT_ERROR, payload: err.response }));
+    };
+};
+
+// TODO NEEDS LOOKING INTO RETURNS UNDEFINED
```
Comment (dionys93) [inline mission-control-client/src/actions/projectActions.js:75]: #20 Empty dataset

File: mission-control-client/src/actions/projectActions.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            .then(res => {
+                const projectGroups = res;
+                console.log(projectGroups)
+                dispatch({ type: GET_PROJECT_GROUPS_SUCCESS, payload: projectGroups })
+            })
+            .catch(err => dispatch({ type: GET_PROJECT_GROUPS_ERROR, payload: err.response }));
+    };
+};
+
+// TODO NEEDS LOOKING INTO RETURNS UNDEFINED
```
Comment (dionys93) [inline mission-control-client/src/actions/projectActions.js:90]: #20 Empty dataset

File: mission-control-client/src/actions/projectActions.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+            .then(res => {
+                const projectGroup = res.data.data.projectGroup;
+                console.log(projectGroup)
+                dispatch({ type: GET_PROJECT_GROUP_SUCCESS, payload: projectGroup })
+            })
+            .catch(err => dispatch({ type: GET_PROJECT_GROUP_ERROR, payload: err.response }));
+    };
+};
+
+// TODO NEEDS LOOKING INTO RETURNS UNDEFINED
```
Comment (dionys93) [inline mission-control-client/src/actions/projectActions.js:105]: #20 Empty dataset

File: mission-control-client/src/actions/projectActions.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+            .then(res => {
+                const members = res.data;
+                console.log(members)
+                dispatch({ type: GET_PROJECT_GROUP_MEMBERS_SUCCESS })
+            })
+            .catch(err => dispatch({ type: GET_PROJECT_GROUP_MEMBERS_ERROR, payload: err.response }));
+    };
+};
+
+// TODO NEEDS LOOKING INTO RETURNS UNDEFINED
```
Comment (dionys93) [inline mission-control-client/src/actions/projectActions.js:120]: #20 Empty dataset

**Review by dionys93 — APPROVED** [review]

File: mission-control-client/src/actions/projectActions.js [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+export const GET_PROJECT_GROUPS_START = 'GET_PROJECT_GROUPS_START';
+export const GET_PROJECT_GROUPS_SUCCESS = 'GET_PROJECT_GROUPS_SUCCESS';
+export const GET_PROJECT_GROUPS_ERROR = 'GET_PROJECT_GROUPS_ERROR';
+
+export const GET_PROJECT_GROUP_START = 'GET_PROJECT_GROUP_START';
+export const GET_PROJECT_GROUP_SUCCESS = 'GET_PROJECT_GROUP_SUCCESS';
+export const GET_PROJECT_GROUP_ERROR = 'GET_PROJECT_GROUP_ERROR';
+
+// ? ALL PROJECT GROUP MEMBERS : [yes - AG] : null
```
Comment (agordon1123) [inline mission-control-client/src/actions/projectActions.js:30]: Can be removed when merging

Comment (agordon1123) [reply mission-control-client/src/actions/projectActions.js:30]: Same with line 35

File: mission-control-client/src/reducers/globalStore.js [hunk 8]
```diff
@@ -1,12 +1,103 @@
...
+            return{
+                ...state,
+                isLoading:false,
+                error: action.payload
+            }
+        case GET_LAMBDA_ROLE_START:
+            return{
+                ...state,
+                isLoading: true,
+                lambdaRole: action.payload
```
Comment (agordon1123) [inline mission-control-client/src/reducers/globalStore.js:86]: Need to remove action.payload from GET_LAMBDA_ROLE_START

**Review by agordon1123 — APPROVED** [review]

**Review by danbergelt — APPROVED** [review]

**Pull request merged**